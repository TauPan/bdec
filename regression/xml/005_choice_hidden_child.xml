<protocol>
    <choice name="c">
        <field name="" length="8" type="integer" value="0" />
        <field name="b" length="8" type="text" />
    </choice>

</protocol>
