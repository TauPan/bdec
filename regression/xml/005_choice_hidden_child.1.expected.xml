<c><b>x</b></c>